app/src/index.ts

```diff
@@ -40,6 +42,22 @@ const configuration: ConfigurationContext = {
   record: Record,
   indyLedgers: selectedLedgers,
   settings: [],
+  customNotification: {
+    component: PersonCredentialScreen,
+    onCloseAction: (dispatch?: React.Dispatch<ReducerAction<any>>) => {
+      if (dispatch) {
+        dispatch({
+          type: BCDispatchAction.PERSON_CREDENTIAL_OFFER_DISMISSED,
+          payload: [true]
```

Comment:
I think its more common in the app to dispatch the object rather than just the value. Able to check. For example:

```
        payload: [{ didDismissSomething: true }],
```